src/components/pages/observation/create/form/index.tsx

```diff
@@ -80,7 +88,19 @@ export default function ObservationCreateForm({ speciesGroups, languages }) {
         facts: Yup.object().required(),
         userGroupId: Yup.array(),
         resources: Yup.array().min(1).required(),
-        terms: Yup.boolean().oneOf([true], "The terms and conditions must be accepted.")
+        terms: Yup.boolean().oneOf([true], "The terms and conditions must be accepted."),
+
+        //custom field data
+        customFields: Yup.array().of(
+          Yup.object().shape({
+            value: Yup.mixed().when("isRequired", {
+              is: true,
+              then: Yup.mixed().required(),
+              otherwise: Yup.mixed().required()
```

Comment:
I this always stays required since is has hardcoded true and otherwise also has same validation as then!